<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResistorCatalog</class>
 <widget class="QDialog" name="ResistorCatalog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLineEdit" name="resistorUID_lineEdit"/>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="resistorR_lineEdit"/>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="resistorAlias_lineEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>R [Ohm]</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QTextEdit" name="resistorDescription_textEdit"/>
   </item>
   <item row="8" column="1">
    <widget class="QDialogButtonBox" name="resistorButtons_buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Rated voltage [kV]</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="resistorX_lineEdit"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>X [Ohm]</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Alias</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>UID</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="resistorName_lineEdit"/>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="resistorRV_comboBox">
     <item>
      <property name="text">
       <string>0.2310000062</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.400000006</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.4199999869</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.5199999809</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3.</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6.</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6.300000191</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10.</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10.38000011</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>New Item</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10.5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>11.</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>13.</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20.</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20.5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>33.</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>35.</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>36.75</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>66.</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>110.</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>132.</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>220.</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>400.</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
